src/aura/STG_CMP_Addr/STG_CMP_AddrController.js

```diff
@@ -2,16 +2,11 @@
     toggleIsView : function(component, event, helper) {
         component.set("v.isView", event.getParam("isView"));
         var enablePrefPhone = component.find("enablePrefPhoneEdit").get("v.value");
```

Comment:
@skosgi1  This line of code can also be removed. Otherwise everything looks good! Feel free to pass this on to Dina once you've made this last change.